src/main/java/seedu/address/ui/MedHistBrowserPanel.java

```diff
@@ -17,9 +17,11 @@
  */
 
 public class MedHistBrowserPanel extends UiPart<Region> {
-
     public static final String DEFAULT_MED_HIST_PAGE = "";
 
+    public static final String MESSAGE_PATIENT_DELETED = "Patient Deleted.";
```

Comment:
Good catch, here you found some "magic numbers" and refactored them into constants.